tests/layer_tests/tensorflow_tests/test_tf_Multinomial.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import pytest
+import tensorflow as tf
+from common.tf_layer_test_class import CommonTFLayerTest
+import numpy as np
+
+
+class TestMultinomial(CommonTFLayerTest):
+    def _prepare_input(self, inputs_dict, kwargs):
+        inputs_dict["num_samples"] = np.array(kwargs["num_samples"], dtype=np.int32)
+        inputs_dict["probs"] = kwargs["input"]
+        return inputs_dict
+
+    def create_tf_multinomial_net(self, global_seed, op_seed, logits_shape, input_type, out_type):
+        tf.compat.v1.reset_default_graph()
+        # Configuration required to make multinomial randomness predictable across devices, results depends on TF parallel execution.
+        session_conf = tf.compat.v1.ConfigProto(intra_op_parallelism_threads=1, inter_op_parallelism_threads=1)
+        # Create the graph and model
+        with tf.compat.v1.Session(config=session_conf) as sess:
+            probs = tf.compat.v1.placeholder(input_type, logits_shape, "probs")
+            num_samples = tf.compat.v1.placeholder(tf.int32, [], "num_samples")
+            if global_seed is not None:
+                tf.random.set_seed(global_seed)
+            tf.random.categorical(tf.math.log(probs), num_samples, dtype=out_type, seed=op_seed)
```

Comment:
please use https://www.tensorflow.org/api_docs/python/tf/raw_ops/Multinomial in the test because we don't know how `categorical` is decomposed and if it contains needed operation

Reply:
Done